src/SarifWeb/Views/Validation/Index.cshtml

```diff
@@ -64,10 +64,10 @@
                     <span class="ui-icon ui-icon-gear">icon</span>
                 </button>
                 <div id="ruleFilters">
```

Comment:
You can fix the text color in CSS by adding an attribute here, such as "no-disable", and then adding this style to wwwroot > css> Validator.css (at line 21):
#validatorToolbar *[no-disable] {
    color: initial;
}

Reply:
just updated!